src/getOldestInstance.ts

```diff
@@ -52,7 +49,14 @@ function getOldestInstanceId(instanceIds: string[]): Promise<Instance> {
     )
 }
 
-class Instance {
+export function findOldestInstance(instances: Instance[]): Instance {
+    let sortedInstances: Instance[] = instances.sort(function(a,b){return a.launchTime.getTime() - b.launchTime.getTime()});
+    let oldestInstance: Instance = sortedInstances[0];
```

Comment:
Minor: could these be `const` ? Same scoping benefits as `let` but indicates that the value will not be changed / re-assigned.

I do have a preference for `const` wherever possible, YMMV 😄